src/Sulu/Bundle/CustomUrlBundle/Resources/public/js/components/webspace/settings/custom-url/overlay/form.html

```diff
@@ -0,0 +1,128 @@
+<form id="custom-url-form" class="grid">
+    <div class="form-group grid-row">
+        <label for="custom-url-title" class="required"><%= translations.title %></label>
+        <input type="text"
+               id="custom-url-title"
+               name="title"
+               class="form-element"
+               data-mapper-property="title"
+               data-validation-required="true"
+               data-validation-min-length="3"/>
+    </div>
+
+    <div class="grid-row">
+        <span class="content">
+            <div data-form="true"
```

Comment:
Do we have `div`s in `span`s more often? The IDE is showing that as an error, and according to a quick google search this is really not allowed.